### PR TITLE
5.7.37 codership mysql wsrep 5.7.36 25.28

### PR DESCRIPTION
**Pull request by nitendra-bhosle**

### CODE REVIEW

File: WSREP_VERSION [hunk 1]
```diff
@@ -0,0 +1,2 @@
+WSREP_VERSION_API=25
+WSREP_VERSION_PATCH=28
```
Comment (venkatesh-prasad-v) [inline WSREP_VERSION]: In `cmake/wsrep.cmake`, we have set `WSREP_PATCH_VERSION=57`, but it is 28 here. This causes inconsistency in versioning. Could you please fix it?

File: cmake/wsrep.cmake [hunk 2]
```diff
@@ -16,29 +16,12 @@
...
-IF (NOT WITH_WSREP)
-  RETURN()
-ENDIF()
-
-FILE(READ "${CMAKE_SOURCE_DIR}/WSREP_VERSION" wVER)
-STRING(REGEX MATCH "WSREP_VERSION_API=([0-9]*)" _ ${wVER})
-SET(WSREP_API_VERSION ${CMAKE_MATCH_1})
-STRING(REGEX MATCH "WSREP_VERSION_PATCH=([0-9]*)" _ ${wVER})
-SET(WSREP_PATCH_VERSION ${CMAKE_MATCH_1})
->>>>>>> wsrep_5.7.36-25.28
```
Comment (venkatesh-prasad-v) [inline cmake/wsrep.cmake]: I think it is better we retain upstream's logic of fetching the `WSREP_PATCH_VERSION` from the WSREP_VERSION file. Could you please revert these changes?

**Review by venkatesh-prasad-v — CHANGES_REQUESTED** [review]

**Review by venkatesh-prasad-v — APPROVED** [review]
lgtm

**Comment by nitendra-bhosle** [comment]
Jenkins run : https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/293/

**Pull request merged**